app/controllers/dashboards_controller.rb

```diff
@@ -4,6 +4,9 @@ class DashboardsController < ApplicationController
 
   def show
     @context = params[:id]
+
+    @active_advertisers = User.advertisers.with_active_campaigns.order(company_name: :asc)
+    @active_campaigns = Campaign.active.order(name: :asc)
```

Comment:
Should we also restrict campaigns to ones currently operating? Its possible to be `active` but not valid for the current day.

Reply:
I don't think so. I think `active` is good enough in this case